tests/tools/onert_train/README.md

```diff
@@ -34,3 +34,62 @@ onert_train \
 ```
 
 `onert_train --help` would help you to set each parameter.
+
+## Example
```

Comment:
This example is for ONE developers, but who are new to the training feature.
I thought that `usage` part isn't enough for the newbie. So I tried to explain a bit more.

But I'm not sure that it is meaningful, or easy to maintain later. So, Any opinion is welcome.